suite/tests/client-interface/file_io.c

```diff
@@ -99,24 +108,53 @@ main()
      */
     /* Test dup/close for stolen FDs. */
     for (int i = rlimit.rlim_max; i < rlimit.rlim_max + DR_STEAL_FDS; i++) {
-        if (dup2(0, i) != -1 || errno != EBADF)
-            fprintf(stderr, "Expected dup2 to return EBADF for stolen FD %d\n", i);
+        if (IF_LINUX_ELSE(i % 2 == 0, true)) {
+            /* Note that on AArch64 dup2 is not available and is automatically replaced
+             * with dup3.
+             */
+            if (dup2(0, i) != -1 || errno != EBADF)
+                fprintf(stderr, "Expected dup2 to return EBADF for stolen FD %d\n", i);
```

Comment:
Generally we use print() to avoid flushing issues with stderr on Windows.  Matters less for output we don't check for but best to be consistent and not have someome use this as a model for output-compared printing.

Reply:
This test uses `fprintf(stderr,...` everywhere. I didn't put much thought into it and just assumed that there was some  reason for it. Do you suggest changing it everywhere in this test?

Reply:
Yes I would replace them all with print().  Xref https://dynamorio.org/page_test_suite.html#autotoc_md272

Reply:
Done.